install-deps

```diff
@@ -8,25 +8,42 @@ set -e
 
 install_openblas() {
     # Get and build OpenBlas (Torch is much better with a decent Blas)
-    cd /tmp/
-    rm -rf OpenBLAS
-    git clone https://github.com/xianyi/OpenBLAS.git
-    cd OpenBLAS
-    if [ $(getconf _NPROCESSORS_CONF) == 1 ]; then
-        make NO_AFFINITY=1 USE_OPENMP=0 USE_THREAD=0
+    # Optionally set environment variable PREFIX to control
+    # the installation directory.
+
+
+    local PATH=$PATH:/sbin ## improve chances of finding ldconfig
+    # Only proceed installing OpenBLAS if either ldconfig is unavailable, or ldconfig
+    # reports that OpenBLAS is not already installed.
+    if ! type ldconfig >/dev/null || ! ldconfig -p | grep -q libblas.so; then
```

Comment:
Thanks for reviewing! I saw that libblas.so.3 is what the official Debian package calls it ( https://packages.debian.org/jessie/amd64/libblas3/filelist ), so I gathered it was the norm so to speak, but perhaps I should add an optional "open" in the grep match expression.